### ISSUE TITLE
Show an Icon representation of the window in the workspace-switcher applet

### ISSUE THREAD

**Issue by anaximeno** [linuxmint/cinnamon#11455]
### Is your feature request related to a problem? Please describe.

When having multiple windows open in many workspaces, it is difficult to know from the workspace-switcher applet, which workspace is the one that contains the window I want to access.

### Describe the solution you'd like

A solution could be to have an option in the config to display the window's icon in the center of its respective frame representation in the applet.

### Describe alternatives you've considered

I currently navigate through all workspaces until I find the workspace that contains the window.

### Additional context

I tried to implement this myself, but until now I did not find any good way to retrieve the Icon of the window object that is used in the applet.

**Comment by rcalixte** [comment]
@anaximeno Do you mean making a view like Expo but in the panel?

**Comment by anaximeno** [comment]
> @anaximeno Do you mean making a view like Expo but in the panel?

Yeah, at least, showing the icon (when available) of the window that is being displayed in the applet.

**Here's what currently happens on Cinnamon**

![Captura de ecrã de 2023-01-31 13-03-11](https://user-images.githubusercontent.com/63073056/215783427-2d36ce88-7533-41cf-91dc-6fd06ab15c81.png)

**Here's an example from XFCE4**

![Captura de ecrã_2023-01-31_13-00-58](https://user-images.githubusercontent.com/63073056/215783700-97fa2c29-7499-4eec-9c78-25fb5a117e3e.png)
  
  
_* I have the same applications open in both examples._
